src/cider/function/scalar/RuntimeFunctions.cpp

```diff
@@ -1480,7 +1480,7 @@ extern "C" ALWAYS_INLINE int32_t extract_str_len_arrow(int8_t* offset_buffer,
 }
 
 #include "exec/nextgen/context/ContextRuntimeFunctions.h"
+#include "exec/nextgen/function/CiderDateFunctions.cpp"
```

Comment:
why these files are included at end of the file?

Reply:
Seems new added function files for nextgen are put here.

Reply:
I think this is just to distinguish which header files (beginning of the file) actually need to be included by `RuntimeFunctions.cpp`, which (end of the file) are new added nextgen function modules. maybe can refactor this file and find better way(like a placeholder header file above `RuntimeFunctions.cpp`) to consolidate these files after fully migrated to nextgen.